src/assessment.js

```diff
@@ -48,5 +55,6 @@ var q2 = [/* your answer here */];
 // method and bind it (or, change `this`) to `alderaan`.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
-};
+   fire.call(alderaan);
```

Comment:
Almost. This won't work because `fire` doesn't exist in the global scope; rather, it's attached to `deathStar`.